bazel/README.md

```diff
@@ -33,9 +33,9 @@ On Ubuntu, run the following command:
 sudo apt-get install \
    libtool \
    cmake \
-   realpath \
-   clang-format-7 \
+   clang-format-6.0 \
```

Comment:
This is very unlikely to work. Formatting changes in each version. You probably need to add instructions on how to install clang-format from the LVM repos like we do in the CI container? https://github.com/envoyproxy/envoy/blob/master/ci/build_container/build_container_ubuntu.sh#L11

Reply:
You're right; that's sad, since it makes it much more awkward for the user. I wonder if we can find a `clang-format-7`-PPA or something...

Reply:
FYI, this tripped me up recently, but Ubuntu 18 just released clang-7 in a normal update, so I think on 18 once can now just install clang-format-7, if you want to update the docs to account for that.

Reply:
Ah, nice! Thanks a lot, I will.

Reply:
There we go @mattklein123, should look more sane now.